### PR TITLE
Address issue 62 - adding task start time

### LINKED ISSUE #62: Consider adding task start time
When deciding whether to opt in to a task, a party knows only when the task expires, not how long the task was in use. Imagine for example a client uploads a report to a task that expires in two years; then two years minus one month later, the Leader wants to aggregate it. Should the Helper oblige?

It sounds reasonable to include in the `TaskConfig` an indication of when the task began.

### PR DESCRIPTION
**Pull request by junyechen1996**
Closes #62.

### CODE REVIEW

**Review by wangshan — APPROVED** [review]

File: draft-ietf-ppm-dap-taskprov.md [hunk 1]
```diff
@@ -197,6 +197,9 @@ struct {
     /* The batch mode and its parameters. */
     opaque batch_config<1..2^16-1>;
 
+    /* Unix epoch time that describes the start time of this task. */
+    Time task_start;
+
     /* Time up to which Clients are allowed to upload to this
     task. */
```
Comment (cjpatton) [inline draft-ietf-ppm-dap-taskprov.md:204]: nit: Consider harmonizing the comments for task_start and task_expiration.

**Review by cjpatton — APPROVED** [review]

**Review by cjpatton — APPROVED** [review]
Thanks for taking initiative here.

**Comment by cjpatton** [comment]
2024/10/24 interim: @junyechen1996 instead of `Time task_start` and `Time task_end`, we will have `Time task_start` and `Duration task_duration`. That way we avoid encoding invalid task validity ranges.

**Comment by junyechen1996** [comment]
> 2024/10/24 interim: @junyechen1996 instead of `Time task_start` and `Time task_end`, we will have `Time task_start` and `Duration task_duration`. That way we avoid encoding invalid task validity ranges.

Updated the PR to reflect this^.

**Review by cjpatton — APPROVED** [review]

**Pull request merged**